emcc.py

```diff
@@ -670,7 +665,7 @@ def phase_parse_arguments(state):
   newargs = diagnostics.capture_warnings(newargs)
 
   for i in range(len(newargs)):
-    if newargs[i] in ('-l', '-L', '-I'):
+    if newargs[i] in ('-l', '-L', '-I', '-z'):
       # Scan for individual -l/-L/-I arguments and concatenate the next arg on
```

Comment:
This comment could also be updated (or just remove the redundant list of flags)

Reply:
Updated comment.